internal/uvm/cpugroups.go

```diff
@@ -10,6 +10,11 @@ import (
 	hcsschema "github.com/Microsoft/hcsshim/internal/hcs/schema2"
 )
 
+// Build that assigning a cpu group on creation of a vm is supported
+const cpuGroupCreateBuild = 20124
+
+var errCPUGroupCreateNotSupported = fmt.Errorf("cpu group assignment on create requires a build of %d or higher", cpuGroupCreateBuild)
+
 // ReleaseCPUGroup unsets the cpugroup from the VM
 func (uvm *UtilityVM) ReleaseCPUGroup(ctx context.Context) error {
```

Comment:
Are we keeping this for the future modify path?

Reply:
Yep